lib/rules/declaration-block-no-redundant-longhand-properties/__tests__/index.js

```diff
@@ -206,6 +206,12 @@ testRule({
 			description: 'autofixer should not mangle css functions with comma separated values',
 			message: messages.expected('transition'),
 		},
+		{
+			code: 'a { border-top-left-radius: 1em; border-top-right-radius: 2em; border-bottom-right-radius: 3em; border-bottom-left-radius: 4em; }',
+			fixed: 'a { border-radius: 1em 2em 3em 4em; }',
+			description: 'explicit border-radius test',
+			message: messages.expected('border-radius'),
+		},
```

Comment:
We will need to add a test for `top-left | top-right-and-bottom-left | bottom-right` and check whether `/` is supported too. 
Thanks for the quick fix.

Reply:
Hm, I think I'm still misunderstanding something. From my understanding, this rule's autofix only looks at cases like

```css
border-top-left-radius: 1em;
border-top-right-radius: 2em;
border-bottom-right-radius: 3em;
border-bottom-left-radius: 4em;
```

and converts them to

```css
border-radius: 1em 2em 3em 4em;
```

If not all four longhand properties are specified, this rule shouldn't report an error (nor try to do an autofix); is that what you mean?

(i.e. this case [from MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius#syntax), which I don't think is "reachable" from our autofix rules)

```css
/* (first radius values) / top-left | top-right-and-bottom-left | bottom-right */
border-radius: 10px 5px 2em / 20px 25px 30%;
```

Reply:
Check #6947 OP, it has a use case for border-width.

e.g. for `border-radius`
```css
border-top-left-radius: 1px;
border-top-right-radius: 4px;
border-bottom-right-radius: 2px;
border-bottom-left-radius: 4px;
```
becomes
```css
border-radius: 1px 4px 2px;
```

Reply:
Oh, I see what you mean. Is that within scope of the rule? Right now, we don't do any "collapsing", i.e. it would output

```css
border-radius: 1px 4px 2px 4px;
```

Reply:
> we don't do any "collapsing"
> Is that within scope of the rule?

It can be a type:enhancement issue for now.
IMHO we should add the cases not covered as tests using `skip: true` and aim to progressively not skip them.

Reply:
Interesting. Let's discuss it with a new issue. Could you open an issue? @Mouvedia